### PR TITLE
[TD-2768]Add phone number in zopim(1)

### PR DESCRIPTION
**Pull request by pranavp99**
### What do you want to achieve?
-Phone number details should be passed to zendesk zopim.

### PR Checklist

- [ ] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-check if entered phone number is similar to the phone contact in zopim user.

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/zendesk-chat-service.js [hunk 1]
```diff
@@ -83,6 +86,15 @@ function ZendeskChatService() {
             });
         }
     };
+    _this.updateNumberInZopim = function() {
+        if(zChat.getVisitorInfo().phone != phoneNumber){
+            zChat.setVisitorInfo({ phone: phoneNumber }, function(err) {
```
Comment (devashishmamgain) [inline webplugin/js/app/zendesk-chat-service.js:91]: this will get called even if the phone number is not provided?

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/zendesk-chat-service.js (hunk 1 above)
Comment (devashishmamgain) [reply webplugin/js/app/zendesk-chat-service.js:91]: so now we will keep calling Zendesk APIs getVisitorInfo even if we have nothing to update

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/zendesk-chat-service.js (hunk 1 above)
Comment (pranavp99) [reply webplugin/js/app/zendesk-chat-service.js:91]: will place an if condition above this to check if phoneNumber exists or not

Comment (pranavp99) [reply webplugin/js/app/zendesk-chat-service.js:91]: only then it'll call getVisitorInfo

Comment (devashishmamgain) [reply webplugin/js/app/zendesk-chat-service.js:91]: read this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND

**Review by devashishmamgain — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Review by gargshantnu — APPROVED** [review]

**Pull request merged**